src/herbie.rkt

```diff
@@ -85,6 +85,11 @@
     (*platform-name* (string->symbol platform))
     (*active-platform* (get-platform (*platform-name*)))
     (activate-platform! (*active-platform*))]
+   [("--old-cost-model")
+    (
+     "Uses the old naive cost model. [Default: New cost model enabled]"
+    )
+    (*old-cost-function* #t)]
```

Comment:
I assume we leave this undocumented?

Also is there a reason the old cost model isn't just another platform? Is number = 0 the only difference?

Reply:
Oh it should be different. I'm making another commit to fix it and add a suggestion Brett made.

Reply:
We probably don't need a command-line option